maven-plugins/pom.xml

```diff
@@ -129,6 +130,22 @@ under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>analyze</id>
+            <goals>
+              <goal>analyze-only</goal>
+            </goals>
+            <configuration>
+              <failOnWarning>${enforce.dependency.declarations}</failOnWarning>
+              <ignoredDependencies>org.apache.maven:maven-compat:*,org.slf4j:slf4j-simple:*</ignoredDependencies>
```

Comment:
should be as separate item for each ignore artifact to allow append more in child project
`ignoredDependencies` is a list parameter

Reply:
last `*` can also be skipped 

https://maven.apache.org/plugins/maven-dependency-plugin/examples/exclude-dependencies-from-dependency-analysis.html

Reply:
good ideas, done